tests/integration/conftest.py

```diff
@@ -359,6 +361,9 @@ async def app_fixture(
         EXTERNAL_BUILD_CONFIG_NAME: "True",
         EXTERNAL_BUILD_FLAVOR_CONFIG_NAME: openstack_metadata.flavor,
         EXTERNAL_BUILD_NETWORK_CONFIG_NAME: openstack_metadata.network,
+        SCRIPT_URL_CONFIG_NAME: "https://raw.githubusercontent.com/canonical/"
+        "github-runner-image-builder/refs/heads/main/tests/integration/testdata/test_script.sh",
```

Comment:
Out of interest how is this script managed?
If the script needs to be changed do we first submit a PR to modify the script, then work with the modified script?

Reply:
Yup - modified script can also be referenced via commit hashes to make it work on a single change PR :)